src/configuration.ts

```diff
@@ -54,6 +54,10 @@ export const getConfig: () => RubocopConfig = () => {
 
   // if executePath is present in workspace config, use it.
   if (configPath.length !== 0) {
+    var delimiter = win32 ? '\\' : '/';
```

Comment:
I believe you can use [`path.sep`](https://nodejs.org/api/path.html#path_path_sep) as the delimiter value here. It frees you from worrying about the Windows version.